### DISCUSSION TITLE
Authorization required on workflow pull request.

### DISCUSSION THREAD

**Discussion by ghost** [vercel/community#5085] [Help]
### Question

Hello! 
I have a question I hope someone may be able to help me with. I have created my first pull request on my personal DAO project on GitHUb. I created the pull request about a week ago, but nothing has happened. I was wondering if maybe I missed a step or have not completed something.  I have no conflicts, the code looks good and the preview on Realms.today looks as expected. Any ideas? or should I just wait a bit longer. It's my first time so I don't know what to expect and I have searched high and low for any information on this topic..

Thanks in advance for any assitance.

**Comment by amyegan** [comment]
Hi @SportBetDAO! Did you already create a Project on Vercel and connect your GitHub repo to it in Project Settings?

https://vercel.com/docs/deployments/git

**Comment by ghost** [comment]
Hi @amyegan ! Thank you for your reply. Yes, as far as I remember I had actually done that, I wasn't sure what to select for the framework preset as it's just a coding upload for a DAO on the Solana Realms.today app. I'll re-read the docs you linked and see if I have missed something. I have re-run it as a new project and got error, which I don't understand, but I will take a look at it. At the moment its still trying to complete the deployment.
Thank you very much for your help, I really appreciate it.

Alex

**Comment by ghost** [comment]
![Screenshot 2023-12-06 at 11 07 09](https://github.com/vercel/community/assets/142555821/7958a325-dc28-46ec-864f-4185c36f33ab)
